source/agora/common/Serializer.d

```diff
@@ -407,22 +407,19 @@ public void serializePart (T) (scope const auto ref T record, scope SerializeDg
                                CompactMode compact = CompactMode.Yes)
```

Comment:
Commit message:

> if the type is some chars, `ubyte`, or `bool`.

Did you mean "is one of: char, ubyte, or bool?

"type is some chars" sounds really strange.

Reply:
[I didn't pick the name](https://dlang.org/phobos/std_traits.html#isSomeChar)

Reply:
I mean that's literally adding whitespace to a function name and calling it English. nono I will send you to English class with Michael & Henry.

Reply:
Fixed